deno.json

```diff
@@ -1,6 +1,6 @@
 {
   "tasks": {
-    "build": "deno run --unstable -A --no-check https://deno.land/x/wasmbuild@0.8.5/main.ts --no-default-features --features wasm",
+    "build": "RUSTFLAGS= deno run --unstable -A --no-check https://deno.land/x/wasmbuild@0.8.5/main.ts --no-default-features --features wasm",
```

Comment:
I removed `RUSTFLAGS: -D warnings` from being used on the CI, because the wasm build was complaining about public methods not being used. I don't think it's worth adding a bunch of `#[allow(unused)]` to these just for Wasm considering the compiler should optimize this away.

Reply:
I noticed the other code was doing `#[cfg(feature = "rust")]` so I just copied that and I'll remove this.